apps/demos/Demos/Chat/Customization/jQuery/styles.css

```diff
@@ -0,0 +1,53 @@
+.dx-viewport {
```

Comment:
I assume this styles are excess, because demo container already have the required styles